### ISSUE TITLE
smooth method broken?

### ISSUE THREAD

**Issue by neishm** [pygeode/pygeode#86]
The unit test for #44 is failing again, though for different reasons.  Running the test code:
```python
import pygeode as pyg
import numpy as np
time = pyg.ModelTime365(values=np.arange(100), units='days', startdate=dict(year=1, month=1))
ts = time.smooth('time', 20)
```
gives the expected `SmoothVar` object, but then trying to load any values (e.g. `ts[0]`, `ts[:]`, or even `ts.load()` results in the message
> AssertionError: Implicit reduction axes (inaxes=None) can only be used when looping over one variable.

Using `git bisect` (first time I used it, but *wow* that's useful) I tracked the problem to somewhere in 808d05f019f4e5d54b96473e5563a14bd4133736.  Peter, I was wondering if you had any insight into what might be happening?

**Comment by aph42** [comment]
Seems to be related to the fact that time is an axis; changing the last line to ts = (2*time).smooth('time', 20) yields a working variable. Not sure yet what's going on but I'll have a look.

**Comment by neishm** [comment]
Yes, you're right.  It works fine on a `Var` object, but not an `Axis`.  After some digging, I traced the bug to `tools.loopover`, nothing to do with `SmoothVar` at all.  If `loopover` is passed a single `Axis` as an argument, it detects a `__len__` attribute and assumes it was passed a list of `Var` objects.  It doesn't take long to crash after that.

I just pushed a patch to fix the real problem (with loopover).  Marking this as "invalid", because it turns out there was no problem with the `smooth` method.
